cpg_infra/billing_aggregator/aggregate/utils.py

```diff
@@ -611,6 +611,8 @@ def upsert_rows_into_bigquery(
     It has some optimisations about max insert size, so this should be
     able to take an arbitrary amount of rows.
     """
+    if not objs:
+        return 0
```

Comment:
There is the same check few lines down, I guess you want to remove that one.

Reply:
Ah yep, good call - thanks :)